build/cmake/CMakeLists.txt

```
@@ -159,6 +159,7 @@ target_link_libraries (glew_s ${GLEW_LIBRARIES})
 
 target_compile_definitions(glew_s INTERFACE "GLEW_STATIC")
 foreach(t glew glew_s)
+  target_compile_definitions(${t} INTERFACE GLEW_NO_GLU)
```

```suggestion
  target_compile_definitions(${t} PUBLIC GLEW_NO_GLU)
```
and remove the directory-scoped definition instead.